_posts/2019/2019-04-02-how-to-easily-manage-ssh-keys-part-2.md

```diff
@@ -0,0 +1,177 @@
+---
```

Comment:
I had some problems following this post. Don't know if it was because I don't have the tech background knowledge.
Would be great if someone from your team can read it through to also check back on context and technical understanding.